lib/STM_thread.ml

```diff
@@ -9,6 +9,8 @@ module Make (Spec: Spec) = struct
 
   exception ThreadNotFinished
 
+  let arb_cmds_conc = arb_cmds_par
```

Comment:
This reads a bit strange, maybe the generic generator `arb_cmds_triple` now that `_par` and `_conc` are used consistently?
(This is arguably only quibbling ;-)

Reply:
Indeed - addressed by renaming to the universal `_triple` suffix